extension.json

```diff
@@ -71,6 +71,12 @@
                 "type": "string",
                 "minLength": 1
               },
+              "clickCollectionEnabled": {
+                "type": "boolean"
+              },
+              "downloadLinkQualifier": {
+                "type": "string"
```

Comment:
What do you think about validating that it's a `minLength` of `1` and adding the respective validation to https://github.com/adobe/reactor-extension-alloy/pull/47/files#diff-9e7d2c5e07f687fc721afa2ef6683586R209? An empty string seems like a mistake that a user would want to know about. I don't think they would want every link to be considered a download, for example.